bears/documentation/DocumentationStyleBear.py

```diff
@@ -49,10 +53,13 @@ def run(self, filename, file, language: str, docstyle: str='default'):
 
             # 1 empty line shall follow main description (except it's empty or
             # no annotations follow).
-            if main_description.desc.strip() != '':
+            if allow_missing_func_desc and main_description.desc.strip() != '':
                 main_description = main_description._replace(
-                    desc='\n' + main_description.desc.strip() + '\n' *
-                         (1 if len(parsed) == 1 else 2))
+                    desc='\n')
+            elif main_description.desc.strip() != '':
+                main_description = main_description._replace(
+                   desc='\n' + main_description.desc.strip() + '\n' *
+                   (1 if len(parsed) == 1 else 2))
```

Comment:
however, why is this a separate commit?  We dont usually split the initial bear into multiple commits.